tests/acceptance/wcag/index.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+
+const puppeteer = require('puppeteer');
+const { AxePuppeteer } = require('@axe-core/puppeteer');
+const PageNavigator = require('./pagenavigator');
+const WcagReporter = require('./wcagreporter');
+const handler = require('serve-handler');
+const http = require('http');
+const PORT = 9999;
+
+const axeCoreConfig = {
```

Comment:
We should add high-level JSDoc explaining this config. Which WCAG standards does it test against? What does 'no-passes' mean?